format/parquet.go

```diff
@@ -148,7 +148,9 @@ type Statistics struct {
 	Max []byte `thrift:"1"`
 	Min []byte `thrift:"2"`
 	// Count of null value in the column.
-	NullCount int64 `thrift:"3"`
+	// Always written (even when zero) to conform to the Parquet spec at
+	// https://github.com/apache/parquet-format/blob/apache-parquet-format-2.12.0/src/main/thrift/parquet.thrift#L283-L291
+	NullCount int64 `thrift:"3,write_zero"`
```

Comment:
Can we name the tag `writezero` for naming consistency (we don't use `_` on any struct tags in parquet-go, nor does the stdlib -- e.g., `omitempty`, `omitzero`).